convert_snapshot.py

```diff
@@ -226,9 +232,20 @@ def bool_to_int(x):
         return 0
 
 
-def obesity_value(x):
-    # The protobuf enum defines NORMAL as 2. We want to treat NOT_APPLICABLE and UNDERWEIGHT as NORMAL.
-    return max(0, x - 2)
+def obesity_value(x, reduce_obesity):
+    """
+    Transform the protobuf obesity enum values for the simulation.
+
+    Always transform NOT_APPLICABLE and UNDERWEIGHT into NORMAL. If
+    reduce_obesity is true, also move people in the obese 1, 2, or 3 categories
+    one level up. Do not transform OVERWEIGHT.
+    """
+    # The protobuf enum defines NORMAL as 2. We want to treat NOT_APPLICABLE
+    # and UNDERWEIGHT as NORMAL.
+    x = max(0, x - 2)
+    if reduce_obesity and x >= 2:
```

Comment:
I would suggest validate this with Karyn, I would imagine she would like to use UNDERWEIGHT not as NORMAL,  just to confirm we got the values they need.

Reply:
Id say we can merge this? No?